pom.xml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is totally not necessary. The Apache Maven super pom upon which ALL Maven poms are extended from does not include an xml declaration, so we should either.
https://maven.apache.org/guides/introduction/introduction-to-the-pom.html